tests/modules/nf-core/deeptools/multibamsummary/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { DEEPTOOLS_MULTIBAMSUMMARY } from '../../../../../modules/nf-core/deeptools/multibamsummary/main.nf'
+
+workflow test_multibamsummary_bam {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
```

Comment:
Question: Did you also test if it works on multiple bam files?

Reply:
Hi, yes I've added the tests for multiple bam files